### ISSUE TITLE
Ingress no address on AWS EKS cluster

### ISSUE THREAD

**Issue by poppahorse** [SocketCluster/socketcluster#498]
So I've got a local k8s cluster setup and have this working fine.

I setup an AWS EKS cluster via Rancher, and the Ingress is stuck on `initializing`. I've been trying to trace what the problem is, and as far as I can see, the ingress has no external address.

```
$ kubectl get ing
NAME          HOSTS   ADDRESS   PORTS   AGE
scc-ingress   *                 80      47m
$ kubectl describe ing scc-ingress
Name:             scc-ingress
Namespace:        default
Address:          
Default backend:  socketcluster:8000 (192.168.184.39:8000)
Rules:
  Host  Path  Backends
  ----  ----  --------
  *     *     socketcluster:8000 (192.168.184.39:8000)
Annotations:
  kubernetes.io/ingress.allow-http:  false
Events:                              <none>
```
On my local cluster, the address shows as the IP's of the nodes.

Also if I do `get pods -n kube-system | grep ingress`, my local cluster shows `rke-ingress-controller-deploy-job-6d5zv` but on EKS there are no resources found.

Also tried `kubectl cluster-info dump | grep ingress` with no response.

Anyone had any experience getting socketcluster working via AWS + EKS / K8s + Rancher?

**TL;DR - Can't seem to determine why the ingress is having issues, either via Rancher, or `kubectl`. Any ideas, or how best to troubleshoot?**

**Comment by happilymarrieddad** [comment]
Umm, we're using it with Kops, AWS and it works well with `ingress-nginx`. I don't know if that's helpful but it does work.
